### ISSUE TITLE
Incorrect use of `deprecated` attribute

### ISSUE THREAD

**Issue by estebank** [vorner/arc-swap#13]
https://github.com/rust-lang/rust/pull/56896 is making the correct use of `deprecated` mandatory in `rustc`. We can't apply this as a hard error due to the following use in `arc-swap`:

```
[INFO] [stderr] error: expected meta item sequence
[INFO] [stderr]    --> /opt/crater/cargo-home/registry/src/github.com-1ecc6299db9ec823/arc-swap-0.3.6/src/lib.rs:500:5
[INFO] [stderr]     |
[INFO] [stderr] 500 |     #[deprecated = "Use upgrade instead"]
[INFO] [stderr]     |     ^^----------------------------------^
[INFO] [stderr]     |       |
[INFO] [stderr]     |       help: use the `note` key: `deprecated(note = "Use upgrade instead")`
```

**Comment by vorner** [comment]
Give me an hour or so, I'll release a fix.

**Comment by vorner** [comment]
A fix should be released in 0.3.7.
